src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +59,18 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+            $resourceShortName = $resourceMetadata->getShortName();
+
+            if (null === $resourceShortName) {
+                throw new InvalidResourceException(sprintf('Resource "%s" has no short name defined.', $resourceClass));
```

Comment:
I added this because there was an error with this test no defining a short name : https://github.com/api-platform/core/blob/master/tests/Bridge/Symfony/Routing/ApiLoaderTest.php#L85

But I do not know if a null short name can happen in a real case.